staging/src/k8s.io/dynamic-resource-allocation/cel/compile.go

```diff
@@ -126,6 +126,9 @@ type Options struct {
 	// DisableCostEstimation can be set to skip estimating the worst-case CEL cost.
 	// If disabled or after an error, [CompilationResult.MaxCost] will be set to [math.Uint64].
 	DisableCostEstimation bool
+
+	// DisableInterrupts can be set to skip checking of the context for cancellation during evaluation.
+	DisableInterrupts bool
```

Comment:
Is it always false? I didn't see where the value is set in a non-test code.

Reply:
It's indeed always false in production. It's useful also without the timeout feature because of the Filter cancellation by the scheduler.

Now that you mention it: perhaps `DisableInterrupts` should get removed entirely? I must have intended to make it depend on the feature gate, but then didn't.